src/main/webapp/app/mood.jsp

```diff
@@ -0,0 +1,68 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: evita.ritina
+  Date: 8/21/2018
+  Time: 17:48
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Feedback Mood page</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://afeld.github.io/emoji-css/emoji.css" rel="stylesheet">
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <style>
+        #Submit {
+            float: right;
+            width: 225px
+        }
+        #Go_back{
+        float: left;
+            width: 130px
+        }
+    </style>
+</head>
+<body>
+<p id="date"></p>
+<form name="submit" id="submit1" action="#" method="post">
```

Comment:
Code has 3 names(id) with "submit"  in your jsp file. Try naming them differently.
For example: "moodForm" "formId" and "submitButton"